plugins/org.eclipse.n4js/src/org/eclipse/n4js/packagejson/projectDescription/ProjectDescription.java

```diff
@@ -279,7 +309,8 @@ public boolean isGeneratorEnabledDts() {
 	@Override
 	protected int computeHashCode() {
 		return Objects.hash(
-				name,
+				id,
```

Comment:
Would be helpful to have a comment above this line, saying that field "location" is deliberately not included (same in method `#computeEquals()`).

Reply:
Thanks for pointing that out. I added the missing fields to the `hashCode` and `equals` methods. Leaving them would be ok somehow but there is no specific reason to do so. Adding them would be at least consistent with the current state, although it could be argued that it is semantically not necessary.